wotw_seedgen/areas.wotw

```diff
@@ -5623,23 +5640,27 @@ anchor InnerWellspring.DrainAreaExit at -1093, -3950:  # At the checkpoint in th
       Spear=2 OR Shuriken=2 OR Blaze=3 OR Flash=2 OR Sentry=2
       Bash  # Bait the skeeto
   conn InnerWellspring.DrainRoom:  # Make sure it's not redundant with DrainAreaExit->DrainRoomCenter->DrainRoom
-    moki: Dash, Glide, Water  # @validator I removed most moki paths because they were redundant with DrainAreaExit->DrainRoomCenter->DrainRoom but there might have some paths using DrainRoomBottomWheel/DrainLever to find here...
+    moki:
+      Dash, Glide, Water OR InnerWellspring.DrainRoomBottomWheel
     gorlek:
+      InnerWellspring.DrainRoomBottomWheel, Glide, Sword OR Hammer
```

Comment:
Not sure what the Hammer variant does exactly. Is it an upswing right before the pole? Gorlek generally doesn't use rapid switches from horizontal movement to up attacks.

Reply:
It was indeed an upswing before the pole, but after retesting it, I agree that it is too hard for Gorlek. Also, I missed it during the validation, but `InnerWellspring.DrainRoomBottomWheel, Dash` looks fine to me for Kii ; and `InnerWellspring.DrainRoomBottomWheel, Glide` works as well (probably unsafe).

Reply:
Sounds good, feel free to add those changes